### DISCUSSION TITLE
Add module "sub_module"

### DISCUSSION THREAD

**Discussion by RealFascinated** [ZoeyVid/NPMplus#579] [Q&A] [answered]
Would you be able to build Nginx with "--with-http_sub_module"? I need to inject html into a site (Grafana) and this seems like the easiest way to achieve that, thanks!

**Comment by Zoey2936** [comment, marked as answer]
I've already added this a few days ago (https://github.com/ZoeyVid/nginx-quic/commit/be282602bf56197c020fa098e014aca324b11a36) since it was needed to add goaccess. So yes, it is already available in the latest release of NPMplus.

**Comment by RealFascinated** [comment]
I was running a older version, just updated and it works. Thank you <3
